UI/question.html

```diff
@@ -0,0 +1,92 @@
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+            </div>
+            <hr>
+            <h1>3 Answers</h1>
+            <div class="answers">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+            </div>
+            <div class="answer">
+                <h1>Your Answer</h1>
+                <form action="">
+                    <label for="uname">
+                        <b>Title</b>
+                    </label>
+                    <input type="text" placeholder="Enter Username" name="uname" required>
+                    <label for="passwd">
+                        <b>Body</b>
+                    </label>
+                    <textarea type="password" placeholder="Enter Password" name="ans" rows="10" cols="50" style="width: 100%" required>
```

Comment:
We where not supposed to use an inline styles

Reply:
Yeah, am still trying to figure out the best way to remove those. I just felt that the css was becoming a bit verbose. Could you recommend some resources about CSS best practices.

Reply:
How about have some utility classes like
```
.m-0{
margin:0;
}
.w-100{
width:100%;
}
```
Which you could then add to your html
Alternatively , have a base styling for those elements
```
textarea{
width:100%;
margin:0;
}
```

Reply:
Great, let me try that👊🏿